src/parser/metrics/Functions.ts

```diff
@@ -19,7 +24,17 @@ export class Functions implements Metric {
 
     async calculate(parseFile: ParseFile, tree: Parser.Tree): Promise<MetricResult> {
         const queryBuilder = new QueryBuilder(parseFile, tree);
-        queryBuilder.setStatements(this.statementsSuperSet);
+        switch (parseFile.language) {
+            case Languages.Java:
+                queryBuilder.setStatements(
+                    this.statementsSuperSet.concat(
+                        new SimpleQueryStatement("(class_body (block)) @initBlock")
+                    )
+                );
+                break;
+            default:
+                queryBuilder.setStatements(this.statementsSuperSet);
+        }
```

Comment:
The same applies here. Refactor to an if-statement.

Reply:
i changed to if else as suggested